drivers/npmx/npmx_shell.c

```diff
@@ -2515,103 +2521,162 @@ static int cmd_gpio_debounce_set(const struct shell *shell, size_t argc, char **
 	return gpio_config_set(shell, argc, argv, GPIO_CONFIG_TYPE_DEBOUNCE);
 }
 
-static int cmd_adc_ntc_get(const struct shell *shell, size_t argc, char **argv)
+static int adc_ntc_get(const struct shell *shell, size_t argc, char **argv,
+		       adc_ntc_config_type_t config_type)
 {
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
-
 	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
-
 	if (npmx_instance == NULL) {
 		shell_error(shell, "Error: shell is not initialized.");
 		return 0;
 	}
 
 	npmx_adc_t *adc_instance = npmx_adc_get(npmx_instance, 0);
 
-	npmx_adc_ntc_type_t ntc_type;
-	npmx_error_t err_code = npmx_adc_ntc_get(adc_instance, &ntc_type);
+	npmx_adc_ntc_config_t ntc_config;
+	npmx_error_t err_code = npmx_adc_ntc_config_get(adc_instance, &ntc_config);
 
 	if (check_error_code(shell, err_code)) {
-		shell_print(shell, "Value: %s.", npmx_adc_ntc_type_map_to_string(ntc_type));
+		uint32_t config_value;
+		switch (config_type) {
+		case ADC_NTC_CONFIG_TYPE_TYPE:
+			if (!npmx_adc_ntc_type_convert_to_ohms(ntc_config.type, &config_value)) {
+				shell_error(shell,
+					    "Error: unable to convert NTC type to resistance.");
+				return 0;
+			}
+			break;
+		case ADC_NTC_CONFIG_TYPE_BETA:
+			config_value = ntc_config.beta;
+			break;
+		}
+
+		shell_print(shell, "Value: %u.", config_value);
```

Comment:
`Ohm` should be printed for `TYPE`.

Reply:
For npmx charger ntc_resistance cold|cool|warm|hot set|get 'Ohm**s**' is printed
https://github.com/NordicSemiconductor/npmx-zephyr/blob/develop/drivers/npmx/npmx_shell.c#L770
https://github.com/NordicSemiconductor/npmx-zephyr/blob/develop/drivers/npmx/npmx_shell.c#L798

Reply:
Changed to `Ohm`

Reply:
@annakielar which is the correct form?